public/global.css in the sveltekit tutorial is named wrong and told to be placed in the wrong place. (#11215)
supabase/supabase: apps/docs/pages/guides/getting-started/tutorials/with-sveltekit.mdx

export const supabase = createClient(env.PUBLIC_SUPABASE_URL, env.PUBLIC_SUPABASE_ANON_KEY)
```

> And one optional step is to update the CSS file `src/routes/style.css` to make the app look nice.
> You can find the full contents of this file [here](https://raw.githubusercontent.com/supabase/supabase/master/examples/user-management/svelte-user-management/src/app.css).

Reviewer: ```suggestion
Optionally, update `src/routes/style.css` with the [CSS from the example](https://raw.githubusercontent.com/supabase/supabase/master/examples/user-management/svelte-user-management/src/app.css).
```

> If you don't add this file, you will need to remove the import statement in `src/routes/+layout.svelte` (created later).

Reviewer: Let's add this as a comment where the import statement is so the user won't forget.
```suggestion
```
